parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -26,6 +26,10 @@ ExecStart=/usr/local/bin/kubelet \
         {{- if IsKubeletConfigFileEnabled}}
         --config /etc/default/kubeletconfig.json \
         {{- end}}
+        {{- if IsKubeletClientTLSBootstrappingEnabled}}
+        --kubeconfig /var/lib/kubelet/kubeconfig \
```

Comment:
should we wait for this fill too in the cse_config.sh because this is required in this code path too?

Reply:
do you mean `wait_for_file /var/lib/kubelet/kubeconfig` in the cse script? Actually we cannot wait for the kubeconfig: we need to let the kublet start up, then communicate with api server to exchange the signed kubeconfig, and finally save to the `--kubeconfig` path. After that, we will get the `/var/lib/kubelet/kubeconfig` ready. Therefore, we don't wait for the kubeconfig, but bootstrap kubeconfig, when TLS bootstrapping feature is enabled.